packages/kbn-apm-synthtrace/src/cli/scenario.ts

```diff
@@ -16,7 +16,15 @@ type Generate<TFields> = (options: {
   range: Timerange;
 }) => SynthtraceGenerator<TFields> | Array<SynthtraceGenerator<TFields>> | Readable;
 
+export interface ScenarioOptions {
+  // Equivalent of APM Server Setting - Transactions.MaxTransactionGroups
+  maxTransactionOverflowCount?: number;
+  // Equivalent of APM Server Setting - Transactions.MaxServices
+  maxServiceOverflowCount?: number;
+}
```

Comment:
What about using the same names as APM Server? - `transactions.max_groups`, `transactions.max_services` and `service_transactions.max_groups`?